Solutions/Corvus.SpecFlow.Extensions/Corvus/SpecFlow/Extensions/Internal/ProcessOutputHandler.cs

```diff
@@ -137,12 +137,10 @@ private void OnOutputDataReceived(object sender, DataReceivedEventArgs e)
         private void OnErrorDataReceived(object sender, DataReceivedEventArgs e)
         {
             string line = e.Data;
```

Comment:
Just to make sure I've understood, have you discovered as a result of adding nullability annotations that `e.Data` will in fact never be null and so you never really needed the test for null that you've now removed?

Reply:
Yes, exactly.